{"uuid": "6fc8e3e2-f44b-4470-9889-38d8a9706750", "children": ["ea7ff546-aeec-468d-a97d-a97127bf8591"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: cannot parse capability: browserVersion\nfrom invalid argument: cannot be empty\nStacktrace:\n#0 0x561d15b9f133 <unknown>\n#1 0x561d158d3966 <unknown>\n#2 0x561d158f0a51 <unknown>\n#3 0x561d158f7551 <unknown>\n#4 0x561d158f01b3 <unknown>\n#5 0x561d15937025 <unknown>\n#6 0x561d15936a11 <unknown>\n#7 0x561d15938386 <unknown>\n#8 0x561d1593818f <unknown>\n#9 0x561d1592f9a3 <unknown>\n#10 0x561d1590446a <unknown>\n#11 0x561d1590555e <unknown>\n#12 0x561d15b5ecae <unknown>\n#13 0x561d15b628fe <unknown>\n#14 0x561d15b6bf20 <unknown>\n#15 0x561d15b63923 <unknown>\n#16 0x561d15b36c0e <unknown>\n#17 0x561d15b86b08 <unknown>\n#18 0x561d15b86c97 <unknown>\n#19 0x561d15b97113 <unknown>\n#20 0x7f9fc7a94b43 <unknown>\n\n", "trace": "  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/andrei/Desktop/code/devops-sandbox/conftest.py\", line 26, in driver\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1684920097701, "stop": 1684920099983}], "start": 1684920097701, "stop": 1684920100010}